backend/Tasque.Core.WebAPI/Program.cs

```diff
@@ -24,11 +25,13 @@
 builder.Services.AddCors();
 
 builder.Services.AddDbContext<DataContext>(
-    o => o.UseNpgsql(builder.Configuration.GetConnectionString("TasqueDb"),
+    o => o.UseNpgsql(builder.Configuration["ConnectionStrings:TasqueDb"], 
         b => b.MigrationsAssembly(typeof(DataContext).Assembly.FullName))
         .EnableDetailedErrors());
 
-builder.WebHost.UseUrls("http://*:5000");
```

Comment:
@Anatolii-Ivanov says that we need this line for deployment

Reply:
Resolved. I've deleted it by mistake.